### ISSUE TITLE
Warning  RetrieveError  11s   cert-manager  Failed to obtain venafi certificate: vcert error: your data contains problems: zone not found

### ISSUE THREAD

**Issue by wallrj** [jetstack/cert-manager-nginx-plus-lab#32]
I followed the README instructions using the new Lab environment which we prepared yesterday with Paul, but I get this.

```
Warning  RetrieveError  11s   cert-manager  Failed to obtain venafi certificate: vcert error: your data contains problems: zone not found

```

```
kubectl describe certificaterequest
Name:         demo-certificate-701839224
Namespace:    default
Labels:       <none>
Annotations:  cert-manager.io/certificate-name: demo-certificate
              cert-manager.io/private-key-secret-name: demo-tls
              kubectl.kubernetes.io/last-applied-configuration:
                {"apiVersion":"cert-manager.io/v1alpha2","kind":"Certificate","metadata":{"annotations":{},"name":"demo-certificate","namespace":"default"...
API Version:  cert-manager.io/v1alpha3
Kind:         CertificateRequest
Metadata:
  Creation Timestamp:  2020-03-27T12:03:45Z
  Generation:          1
  Owner References:
    API Version:           cert-manager.io/v1alpha2
    Block Owner Deletion:  true
    Controller:            true
    Kind:                  Certificate
    Name:                  demo-certificate
    UID:                   060e9dcb-a76a-4bec-8b49-058323dc66bb
  Resource Version:        7567
  Self Link:               /apis/cert-manager.io/v1alpha3/namespaces/default/certificaterequests/demo-certificate-701839224
  UID:                     829371f9-e7c6-43cb-a056-75819408fa92
Spec:
  Csr:  LS0tLS1CRUdJTiBDRVJUSUZJQ0FURSBSRVFVRVNULS0tLS0KTUlJQ2lqQ0NBWElDQVFBd0Z6RVZNQk1HQTFVRUNoTU1ZMlZ5ZEMxdFlXNWhaMlZ5TUlJQklqQU5CZ2txaGtpRwo5dzBCQVFFRkFBT0NBUThBTUlJQkNnS0NBUUVBb0k3a3M0VlRucm5ENndPYlNyTktMYzA5bStFaVNtclBnUDFrCkJua2RLTXRvdXQ0NThrNXNHd0RkMmZrSGp0MmRoOFFyKzRsMUxqUVlwQm0rR3I0NHR1ZkNhS2c2YjJRZlZuajIKTjR3SGdURm1ORXhuZDAyQWQ0RVZzSDJKbm04WERRM0NpbVZBRkJMeDBYZ3JBZWlpWFIxZEFHUStudG5xYXF3QQpTL0lHQXB2b1l4UXNHV3IrcWhFaE1peWVQRVp6WlV5bHY2ckltSHdrNlBRTWZYaXgxUSt2bm9UWnpyRE5RV011ClhveUNDMXRjV1NTeEFJYll0NUdON2o3ZllZYTdTV1V6ckZ4bGN5RkxEYUp4OUtua29sY0RrUjF3ajRuMXA5TEcKUENYQWd5L05oNFFSMVRNSnBqMEJNcVZzMDJ5M0ROYzVhdk5xRnM5aFM4S3owdXJTUlFJREFRQUJvQzR3TEFZSgpLb1pJaHZjTkFRa09NUjh3SFRBYkJnTlZIUkVFRkRBU2doQmtaVzF2TG1WNFlXMXdiR1V1WTI5dE1BMEdDU3FHClNJYjNEUUVCQ3dVQUE0SUJBUUFOaDFrdlMzeCtTOTVDclZ1eWNUTWROb0dDeEg1ODBxZXFaOVQyNkRpVDFYVjkKb3RDU2dLNXJXQWtPam9UM1NvMDgvMHBDNVZyTkcxZ0pxYTlMbnAyV1hPc2FXbXc0YzVGcGhlc09MRGtDWDZRVwplRy9KYjVBUU5vQlgya3ZxZytpd1ExVm5pS1FLcTArVkVsV3BYVmRGc1g0WWpqeS9XYWNtWU5BOHB3Y2hoT2dFCnBHWGprUWpoaGhwai9XQ3liam5lUzBVdFp3YzBaWVBMOVFTSTA1b1JsYUVHVUQrRGlWbGpVUE1MOXVHR1dackwKVzF6dzMzWVQrTWYvYlNWL1U0T1RWVjJqdXhQSTNNZG8xUmhmYjFKSGpveEZuUURnYktDalgyV2V1Z0dUYy9LSwowclR4VmdVeWVNRDJ1eTIrMmV1TGZqU09kREF6bUppbUthTWF0ckJFCi0tLS0tRU5EIENFUlRJRklDQVRFIFJFUVVFU1QtLS0tLQo=
  Issuer Ref:
    Kind:  Issuer
    Name:  venafi-tpp-issuer
Status:
  Conditions:
    Last Transition Time:  2020-03-27T12:03:46Z
    Message:               Failed to obtain venafi certificate: vcert error: your data contains problems: zone not found
    Reason:                Failed
    Status:                False
    Type:                  Ready
  Failure Time:            2020-03-27T12:03:46Z
Events:
  Type     Reason         Age   From          Message
  ----     ------         ----  ----          -------
  Warning  RetrieveError  11s   cert-manager  Failed to obtain venafi certificate: vcert error: your data contains problems: zone not found

```

**Comment by meyskens** [comment]
The zone we configured was for our test TPP not the one we will use in the new lab. Gonna PR a fix soon, happy to chat to explain how it works as it isn't very obvious

**Comment by meyskens** [comment]
Fixed in #34
